assets/js/type-defs/registered-payment-method-props.js

```diff
@@ -119,20 +119,29 @@
 /**
  * @typedef BillingDataProps
  *
- * @property {CartBillingAddress} billingData    The address used for billing.
- * @property {Function}           setBillingData Used to set the cart billing
- *                                               address.
- * @property {Object}             order          The order object for the
- *                                               purchase.
- * @property {boolean}            orderLoading   True if the order is being
- *                                               loaded.
- * @property {CartTotalItem}      cartTotal      The total item for the cart.
- * @property {SiteCurrency}       currency       Currency object.
- * @property {string}             country        ISO country code for the
- *                                               default country for the site.
- * @property {CartTotalItem[]}    cartItems      The various subtotal amounts.
- * @property {string[]}           appliedCoupons All the coupons that were
- *                                               applied.
+ * @property {CartBillingAddress} billingData               The address used for
+ *                                                          billing.
+ * @property {Function}           setBillingData            Used to set the cart
+ *                                                          billing address.
+ * @property {Object}             order                     The order object for
+ *                                                          the purchase.
+ * @property {boolean}            orderLoading              True if the order is
+ *                                                          being loaded.
+ * @property {CartTotalItem}      cartTotal                 The total item for
+ *                                                          the cart.
+ * @property {SiteCurrency}       currency                  Currency object.
+ * @property {string}             country                   ISO country code
+ *                                                          for the default
+ *                                                          country for the
+ *                                                          site.
+ * @property {CartTotalItem[]}    cartItems                 The various subtotal
```

Comment:
I agree that naming is important.

- I'm meh on using "line" or "row" because these are not lines or rows, they are numerical values. "Line" or "row" might be fine in the naming of a component because the value would be contained within a structure for that "line" or "row".
- "Subtotals" seems more apt, but in the REST api schema, data store, and current components "totals" is used (aside from one use of `SubtotalsItem` in a component. So currently, consistency wise, it seems prudent to continue the "total" naming here.
- With that said, I _do_ think that `cartItems` for the property name is a bad choice here. So I'll rename that to `cartTotalItems` so that it is consistent with the typedef naming.